### DISCUSSION TITLE
Formato de entrega ejercicio Macowins

### DISCUSSION THREAD

**Discussion by AlejoGol** [dds-jv/foro#337] [Consultas generales] [answered]
Hola, buenas
Tengo dudas sobre como es la entrega del ejercicio Macowins que nos dieron para esta semana.
Entiendo que en el formulario de entrega hay que poner el link al repo de github, pero las dudas vienen mas por el lado de lo que va en el repositorio.
Para el diagrama de clases hay que subir una imagen de lo hecho (sin importar la herramienta) o hay otra forma?
Y para el pseudocodigo hay que hacer un proyecto de java (o wollok) y meter el proyecto en el repo (ademas del diagrama, obvio)? o hay que subir solo un archivo de algun formato especifico con el pseudocodigo (considerando que al ser pseudocodigo deberia ser una explicacion del diagrama que no necesita compilar ni necesariamente deberia estar todo de todo)?
Muchas gracias

**Comment by gastonprieto** [comment, marked as answer]
Buenas!

Te respondo: 

> Para el diagrama de clases hay que subir una imagen de lo hecho (sin importar la herramienta) o hay otra forma?

Si, una imagen es lo mas  facil

>Y para el pseudocodigo hay que hacer un proyecto de java (o wollok) y meter el proyecto en el repo (ademas del diagrama, obvio)? o hay que subir solo un archivo de algun formato especifico con el pseudocodigo (considerando que al ser pseudocodigo deberia ser una explicacion del diagrama que no necesita compilar ni necesariamente deberia estar todo de todo)?

la que hayas hecho, puede ser código java/wollok o pseudocódigo, mi recomendación es que vayas haciendo  pseudocódigo asi te vas acostumbrando para el parcial y sabes cuanto detalle poner o no en ese pseudocódigo

**Reply by AlejoGol** [reply]
Okey, entonces puedo hasta subir un archivo .txt con las clases y siempre que este bien explicado estaria bien?

**Reply by gastonprieto** [reply]
si, correcto. Si lo haces en github podes tambien poner un .md q te deja darle formato a las cosas (opcional) te dejo la documentación por si te sirve: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

**Reply by AlejoGol** [reply]
Perfecto, muchas gracias.
Y, perdon, una cosa mas, hay algun codigo para el highlight del codigo en wollok? o sea, de la forma en la que para codigo java despues de los primeros ``` se pone java, hay una palabra para wollok? porque no la estaria encontrando entre la documentacion.

**Reply by gastonprieto** [reply]
por ahora no, el de java nomas, pero sirve para el de wollok ese o el de javascript, lo formatean dentro de todo potable

**Reply by AlejoGol** [reply]
okok, perfecto. Muchas gracias
